### PR TITLE
Output Python floats

### PR DESCRIPTION
**Pull request by MaxHalford**

### CODE REVIEW

**Comment by smastelini** [comment]
Hi @MaxHalford and @agriyakhetarpal, I will try to look at the remaining changes during the hours between my commutes. For simplicity, I will use the same branch Max used and leverage this PR.

File: river/optim/initializers.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ class Normal(Initializer):
     >>> init = optim.initializers.Normal(mu=0, sigma=1, seed=42)
 
     >>> init(shape=1)
-    0.496714
```
Comment (smastelini) [inline river/optim/initializers.py:83]: Should we use `.item()` on this one too?

File: river/proba/gaussian.py [hunk 2]
```diff
@@ -207,7 +207,7 @@ class MultivariateGaussian(base.MultivariateContinuousDistribution):
     >>> multi.mu['blue'] == single.mu
     True
     >>> multi.sigma['blue']['blue'] == single.sigma
-    True
```
Comment (smastelini) [inline river/proba/gaussian.py:210]: Does this need a casting to Python's bool?

**Comment by smastelini** [comment]
All the tests pass on my local setup using OSX and Python 3.11 (I've also tested using 3.12).

**Review by smastelini — COMMENTED** [review]
There are two situations where np objects are explicitly returned. Should we change those too?

**Comment by MaxHalford** [comment]
No it's all good. It's expected that the initializers output arrays.

**Comment by smastelini** [comment]
Cool!

**Comment by agriyakhetarpal** [comment]
Hi all, thank you so much for the PR! I also mostly agree with initialisers returning arrays.

**Pull request merged**